exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/WebUserConnection.java

```diff
@@ -106,7 +116,25 @@ public void sendData(RpcOutcomeListener<Ack> listener, QueryWritableBatch result
         // TODO:  Clean:  DRILL-2933:  That load(...) no longer throws
         // SchemaChangeException, so check/clean catch clause below.
         for (int i = 0; i < loader.getSchema().getFieldCount(); ++i) {
-          columns.add(loader.getSchema().getColumn(i).getName());
+            MaterializedField col = loader.getSchema().getColumn(i);
```

Comment:
What about varchar? This type has length...

Reply:
This change was done for code cleanup.   Since I was referring to the `column` several times, it was easier to create a `col` variable.